health/notifications/alarm-notify.sh.in

```diff
@@ -589,6 +594,9 @@ filter_recipient_by_criticality() {
 # check matrix
 { [ -z "${MATRIX_HOMESERVER}" ] || [ -z "${MATRIX_ACCESSTOKEN}" ]; } && SEND_MATRIX="NO"
 
+# check gotify
+[ -z "${GOTIFY_APP_TOKEN}" ] && SEND_GOTIFY="NO"
```

Comment:
Should we also check "GOTIFY_APP_URL" here?

Reply:
Sure